chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -297,6 +297,23 @@
                         !node[:provisioner][:suse][:cloud_available] && node[:provisioner][:suse][:storage_available],
                       crowbar_join: "#{os_url}/crowbar_join.sh")
           end
+          template "#{node_cfg_dir}/autoyast-upgrade.xml" do
+            mode 0644
+            source "autoyast-upgrade.xml.erb"
+            owner "root"
+            group "root"
+            variables(
+                      admin_node_ip: admin_ip,
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.